config/keys.js

```diff
@@ -7,11 +9,13 @@ module.exports = {
   },
   googleAuth: {
     googleClientId: process.env.GOOGLE_CLIENTID,
-    googleClientSecret: process.env.GOOGLE_CLIENTSECRET
+    googleClientSecret: process.env.GOOGLE_CLIENTSECRET,
+    callbackURL: '/auth/google/callback' 
   },
   facebookAuth: {
-    facebookClientId: process.env.FACEBOOK_CLIENTID,
-    facebookClientSecret: process.env.FACEBOOK_CLIENTSECRET
+    facebookClientId: '482046309075252',
```

Comment:
we cannot store keys in code - this is a gross violation
please, remove keys from here and add it to Heroku env!

make the same one operations on contest side

Reply:
I think Isla had stored the key in code. Okay, I will remove it.

Reply:
You definitely should get keys from process.env.somekey

Reply:
don't forget that we have separate env for each server (master, deployment, contest)